docs/self-managed/setup/deploy/other/docker.md

```diff
@@ -8,21 +8,22 @@ keywords: ["camunda docker"]
 While the Docker images themselves are supported for production usage, [Docker Compose](/self-managed/setup/deploy/local/docker-compose.md) files are designed to be used by developers to run an environment locally; they are not designed to be used in production. We recommend to use [Kubernetes](/self-managed/setup/install.md) in production.
 :::
 
-We provide Docker images [via Dockerhub](https://hub.docker.com/u/camunda). All these images are publicly accessible (except for [Web Modeler](#web-modeler)).
+We provide Docker images [via Dockerhub](https://hub.docker.com/u/camunda). All these images are publicly accessible. Web Modeler consists of three components: `restapi`, `webapp`, and `websockets` which are publicly accessible for versions `>=8.6.0-alpha2` and for enterprise customers only all versions before.
```

Comment:
❓ Do we have to mention this at all here? 

The documentation is versioned so users who want to install `8.5` would directly access the `8.5` section of the documentation still stating that Web Modeler is not publicly available. For `next`, I'd say we don't have to explicitly mention that `8.6-alpha1` is not publicly available - the need for installing old alpha versions should be very low.